transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/VertxRestTransport.java

```diff
@@ -59,7 +59,7 @@ public boolean canInit() {
     }
 
     if (!NetUtils.canTcpListen(ep.getSocketAddress().getAddress(), ep.getPort())) {
-      log.info("can not listen {}, skip {}.", ep.getSocketAddress(), this.getClass().getName());
+      log.warn("can not listen {}, port may have been occupied, this may cause client report error when being accessed. skip {}.", ep.getSocketAddress(), this.getClass().getName());
```

Comment:
so strange......
canTcpListen is to detect, if this transport can init
when customers deployed vertx and servlet rest transport, we must detect which one will be used
only both of them refused to init is a problem.
we have already print that......

Reply:
We can't detect if the started port is tomcat, so in this scenario(port started by another service, and the dependency contains both transport-rest-servlet and transport-rest-vertx) will cause problem. May the warning message can be changed as: 
WARNING: Can not start VertxRestTransport, the port may have been occupied. You can ignore this message if you are using a web container like tomcat.

Reply:
done